.github/workflows/build-push-image-stage.yml

```diff
@@ -0,0 +1,30 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.stage'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Build and push Image
+        uses: Pearson-Advance/tutor-build-image-action@0.1.1
+        with:
+          python-version: ${{ vars.BUILD_PYTHON_VERSION }}
+          tutor-version: ${{ vars.BUILD_TUTOR_VERSION }}
+          path-image-to-be-built: '/env/build/openedx'
+          repository-name: ${{ vars.BUILD_STAGE_REPOSITORY_NAME }}
+          image-name: ${{ vars.BUILD_STAGE_IMAGE_NAME }}
+          image-tag: ${{ vars.BUILD_STAGE_IMAGE_TAG }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-token: ${{ secrets.DOCKERHUB_TOKEN }}
+          no-docker-cache: true
```

Comment:
this disables the caching capabilities. Is that desirable?

Reply:
@alexjmpb thanks for notice it.

Reply:
@alexjmpb suggestion applied.